webpack.config.js

```diff
@@ -0,0 +1,915 @@
+const path = require('path');
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const MiniCssExtractPlugin  = require('mini-css-extract-plugin');
+const CopyPlugin = require('copy-webpack-plugin');
+const process = require('process');
+
+function resolveBundles(fileList){
+  return fileList.map((f) => {
+    if (f.match(/^protocol\//)) {
+      return `@mozilla-protocol/core/${f}`;
+    }
+    return path.resolve(__dirname, "media", f);
+  });
+}
+
+const entry = {
+  "about": resolveBundles([
```

Comment:
I wonder if instead of creating a new list that needs to be updated manually, we could import `static-bundles.json` and then traverse that here? Either that, or write a script convert one format to the other? Either way, separating this from the webpack config itself may be worthwhile.